README.md

```diff
@@ -19,5 +19,5 @@
 
 ## Usage
 
-1. Copy the contents of you're prefered style (ex. `mocha.conf`) into `$HOME/.config/conky/<stylename>.conf`.
-2. Run using `conky -c $HOME/.config/conky/<stylename>.conf`
+1. Copy the contents of your preferred style (ex. `mocha.conf`) into `$HOME/.config/conky/<flavour>.conf`.
+2. Run using `conky -c $HOME/.config/conky/<flavour>.conf`
```

Comment:
```suggestion
1. Copy the contents of your preferred style (ex. `mocha.conf`) into `$HOME/.config/conky/<flavor>.conf`.
2. Run using `conky -c $HOME/.config/conky/<flavor>.conf`
```

Reply:
This is a matter of British vs American spelling and is outside of the scope of this PR.

Reply:
We typically use the American spelling for consistency across Catppuccin repos, but yeah it was just a suggestion not a requirement.